.github/workflows/test.yaml

```
@@ -26,6 +26,7 @@ jobs:
         run: cargo fmt -- --check
       - name: build-enclone
         run: cargo build
```

```suggestion
        run: cargo build --release
```
`cargo build` and `cargo test` need to both use `--release` or neither.